ui/app/AppLayouts/Wallet/stores/RootStore.qml

```diff
@@ -39,8 +39,13 @@ QtObject {
     property var overview: walletSectionOverview
     property bool balanceLoading: overview.balanceLoading
     property var accounts: walletSectionAccounts.accounts
-    property var receiveAccounts: walletSectionSend.accounts
-    property var selectedReceiveAccount: walletSectionSend.selectedReceiveAccount
+    property var selectedReceiveAccount: {
```

Comment:
`selectedReceiveAccount` -> `selectedReceiverAccount `
basically everywhere `%receiveAccount%` should be `%receiverAccount%`